packages/terra-form-textarea/src/Textarea.scss

```diff
@@ -0,0 +1,151 @@
+:local {
+  .textarea {
+    background-color: var(--terra-form-textarea-background-color, #ffffff);
+    background-image: var(--terra-form-textarea-focus-animation-background, none);
+    background-position: 0 100%;
+    background-repeat: no-repeat;
+    background-size: var(--terra-form-textarea-focus-background-start, 0 0);
+    border-color: var(--terra-form-textarea-border-color, #dedfe0);
+    border-radius: var(--terra-form-textarea-border-radius, 3px);
+    border-style: var(--terra-form-textarea-border-style, solid);
+    border-width: var(--terra-form-textarea-border-width, 1px);
+    color: var(--terra-form-textarea-color, #000);
+    cursor: var(--terra-form-textarea-cursor, auto);
+    font-family: var(--terra-base-font-family, 'Helvetica Neue', Helvetica, Arial, sans-serif);
```

Comment:
Why is font-family being set here? Shouldn't this be inherited from font-family being set on the [HTML element](https://github.com/cerner/terra-core/blob/master/packages/terra-base/src/Base.scss#L8)?

Reply:
What we are finding is that browsers are setting the font-family on inputs and textareas.

````
    font-family: system-ui;
````

After showing to UX, they wanted to set the value of font-family to the same value as --terra-base-font-family. Our options are to either create a new theming variable for textarea and default it to the value of terra-base-font-family, or include the base variable inside the textarea. I feel like the second approach is better as it enforces the textarea text to be the same as the base text.

Reply:
Let's test setting this to `font-family: inherit;` This should override the browser default styles that render text inside form elements as that of the user's operating system font while also inheriting the font-family from the HTML provided font-family isn't set anywhere on parent elements between the textarea and the HTML element.

My concern with the current code is, if we need to update hard-coded default font, we need to remember all the instances we set it as a fallback and update those. If we set font-family to inherit, it should achieve more or less the same look while allowing us maintain 1 place to change the font-family used throughout all the components.

Reply:
It works. https://github.com/cerner/terra-core/pull/982/commits/75bb0887272e5d1c74f482490b20e6d5275916b3